ruby_on_rails/recaptcha.md

```diff
@@ -1,35 +1,43 @@
 # reCAPTCHA v3
 
-Here you will learn how to implement reCAPTCHA v3 into a project. It is pretty simple to use and doesn't hinder the users at all, as it all runs in the background.
+Here you will learn how to implement reCAPTCHA v3 into a project. It is pretty simple to use and doesn't
+hinder the users at all, as it all runs in the background.
```

Comment:
I dislike random whitespace cleanup cluttering up pull requests. Maybe we should use common editor settings for `.md` files?